health-on-time/src/Pages/AddMedication.js

```diff
@@ -1,9 +1,62 @@
-import React from "react";
+import React, { useState } from "react";
+import SearchMedication from '../Components/SearchMedication';
+import { Container } from 'react-bootstrap';
+import MedSchedulerMain from '../Components/MedScheduler/MedSchedulerMain.js';
+
+
+
+const AddMedication = ({ handleNewPrescription, prescriptions, token }) => {
```

Comment:
not useContext ?

Reply:
Yes. You're right. Made this change as well as FinalStep.js was using props.

Reply:
Will commit soon

Reply:
done commits and pushed these changes